packages/main/src/Dialog.js

```diff
@@ -390,6 +402,8 @@ class Dialog extends Popup {
 	}
 
 	_attachResizeHandlers() {
+		ResizeHandler.deregister(this, this._screenResizeHandler);
+
```

Comment:
Add also 
ResizeHandler.deregister(document.body, this._screenResizeHandler);

Reply:
Done. Also did the same for the Drag events. Now, the webcomponents acts the same as the OpenUI5 Dialog control with regards to the above comments.